contrib/init/systemd/docker.service

```diff
@@ -5,7 +5,8 @@ After=network.target docker.socket
 Requires=docker.socket
 
 [Service]
-ExecStart=/usr/bin/docker -d -H fd://
+EnvironmentFile=/etc/default/docker
```

Comment:
@jfrazelle I saw the documentation change, thank you.  The 1.7 release notes haven't been updated though, so people will keep running into this problem without warning.